packages/wallet/src/types.ts

```diff
@@ -0,0 +1,67 @@
+import { DefaultWallet, abi } from '@pigi/core'
+
+export type UniTokenType = 'uni'
+export type PigiTokenType = 'pigi'
+export type TokenType = UniTokenType | PigiTokenType
+
+export type SUCCESS = 'SUCCESS'
+export type FAILURE = 'FAILURE'
+
+export type Address = string
+
+export interface Balances {
+  [tokenType: string]: number
```

Comment:
I think you can make this type `TokenType` to restrict keys to valid `TokenType` strings